src/api/network.rs

```diff
@@ -138,8 +138,8 @@ impl NetworkApi {
                 peer_list
                     .iter()
                     .map(|p| Peer {
-                        peer_id:  p.node_id.to_string(),
-                        metadata: Some(json!({ "ip": p.ip, "port": p.port })),
+                        peer_id:  p.peer_id.0.clone(),
```

Comment:
Why is the `0` necessary?

Reply:
It seems `PeerId` is a tuple struct containing only a single `String` element. I imagine it was implemented this way so that you could represent `PeerId` as its own type rather than just a string, such that the compiler can statically check it.